### PR TITLE
Message serializer

### PR DESCRIPTION
**Pull request by bkmaibach**
What the user enters now becomes an entry (formerly message). This is to prevent confusion with the messages that the websocket may send, which may contain several entries.

This facilitates the creation of Two serializers, EntrySerializer and MessageSerializer, which may be nested in the future. Currently, only MessageSerializer is used because the saving of an entry to the database requires looking up the token and auto timestamp, which complicates things.

### CODE REVIEW

File: apps/chatter/classes.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Entry(object):
+    def __init__(self, text, timestamp, author, room):
+        self.text = text
+        self.timestamp = timestamp
+        self.author = author
+        self.room = room
```
Comment (staydecent) [inline apps/chatter/classes.py:6]: What is the purpose of having an Entry model and an Entry class?

File: apps/chatter/consumers.py [hunk 2]
```diff
@@ -4,8 +4,11 @@
 from rest_framework.authtoken.models import Token
 from django.contrib.auth.models import AnonymousUser
 from channels.db import database_sync_to_async
-from .models import Message
+from .models import Entry
 from .models import Room
+from asgiref.sync import sync_to_async
```
Comment (staydecent) [inline apps/chatter/consumers.py:9]: Please follow pep8 style guide (I'll look into adding linting to django-api-starter) https://www.python.org/dev/peps/pep-0008/#imports

File: apps/chatter/consumers.py [hunk 3]
```diff
@@ -88,31 +90,43 @@ async def error_reponse(self, error):
...
+                await self.error_reponse('No procedure is available for command \"{0}\"'
+                    .format(validMessage['command']))
         else:
-            await self.error_reponse('The command \"{0}\" could not be recognized'
-                .format(data['command']))
+            await self.error_reponse('The data sent could not be recognized')
+            print('INVALID SERIALIZER: ', message_serializer.errors)
+        
 
+    async def send_message(self, message):
```
Comment (staydecent) [inline apps/chatter/consumers.py:123]: You have `async def send_message` twice.

File: apps/chatter/models.py [hunk 4]
```diff
@@ -1,21 +1,21 @@
 from django.db import models
 from django.utils import timezone
 from django.conf import settings
+from channels.db import database_sync_to_async
```
Comment (staydecent) [inline apps/chatter/models.py:4]: Unused import.

File: apps/chatter/views.py [hunk 5]
```diff
@@ -7,25 +7,8 @@
 
 class RoomViewSet(viewsets.ModelViewSet):
     """API endpoint that allows rooms to be viewed or edited."""
+    authentication_classes = (TokenAuthentication,)
```
Comment (staydecent) [inline apps/chatter/views.py:10]: This is redundant, as it's the default: https://github.com/inputlogic/chatter-api/blob/2ee810e5c98019a1828ea3830f1652664f785f31/project/settings.py#L223

File: project/settings.py [hunk 6]
```diff
@@ -125,8 +130,8 @@ def get(variable):
         'default': {
             'ENGINE': 'django.db.backends.postgresql',
             'NAME': 'chatter',
-            'USER': 'django',
-            'PASSWORD': 'P@ssword1'
+            'USER': 'postgres',
```
Comment (staydecent) [inline project/settings.py:133]: haha, didn't have to do this, but it is appreciated 🙇

**Review by staydecent — CHANGES_REQUESTED** [review]
Nice work using DRF serializers! It's probably overkill in this case, but good practice to get more familiar with DRF.

Couple small issues noted below.

File: apps/chatter/classes.py (hunk 1 above)
Comment (bkmaibach) [reply apps/chatter/classes.py:6]: Good question! I was experimenting with some of the examples on serializers trying to figure out what made the most sense. Removing!

File: apps/chatter/consumers.py (hunk 3 above)
Comment (bkmaibach) [reply apps/chatter/consumers.py:123]: I shall vanquish this

File: apps/chatter/views.py (hunk 5 above)
Comment (bkmaibach) [reply apps/chatter/views.py:10]: Ah! Good to know.

File: project/settings.py (hunk 6 above)
Comment (bkmaibach) [reply project/settings.py:133]: Not a problem! Good postgres practice 👌

File: apps/chatter/consumers.py (hunk 2 above)
Comment (bkmaibach) [reply apps/chatter/consumers.py:9]: Am I STL making this mistake?
S tandard libraries
T hird party imports
L ocal project imports

There, remembered forever now :D

File: apps/chatter/models.py (hunk 4 above)
Comment (bkmaibach) [reply apps/chatter/models.py:4]: Indeed. Gone now!

**Review by staydecent — APPROVED** [review]

**Pull request merged**